ci/terraform/oidc/authdev1.tfvars

```diff
@@ -8,12 +8,14 @@ account_intervention_service_action_enabled = true
 account_intervention_service_abort_on_error = true
 send_storage_token_to_ipv_enabled           = true
 call_ticf_cri                               = true
-ipv_backend_uri                             = "https://ipvstub.signin.authdev1.dev.account.gov.uk"
+ipv_jwks_call_enabled                       = true
```

Comment:
Despite this being enabled, I think calling the JWKS endpoint may actually fail because it times out from a cold start.